bsp_diff/caas/device/intel/civ/host/vm-manager/0046-Multi-Android-VM-support.patch

```diff
@@ -0,0 +1,382 @@
+From 9549350b8ad6236b8d334aa122e1bd0265cf2ad1 Mon Sep 17 00:00:00 2001
+From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
+Date: Thu, 15 Jun 2023 12:13:48 +0530
+Subject: [PATCH] Multi Android VM support
+
+Tracked-On: OAM-110639
+
+Signed-off-by: Suresh, Prashanth <prashanth.suresh@intel.com>
+---
+ scripts/setup_civ_ini.sh      |  47 ++++++++++--
+ scripts/setup_host.sh         |  51 ++++++++++--
+ scripts/setup_multi_civ_vm.sh | 141 ++++++++++++++++++++++++++++++++++
+ scripts/start_flash_usb.sh    |  22 +++++-
+ 4 files changed, 243 insertions(+), 18 deletions(-)
+ create mode 100755 scripts/setup_multi_civ_vm.sh
+
+diff --git a/scripts/setup_civ_ini.sh b/scripts/setup_civ_ini.sh
+index aab7697..64760eb 100755
+--- a/scripts/setup_civ_ini.sh
++++ b/scripts/setup_civ_ini.sh
+@@ -8,29 +8,38 @@
+ set -eE
+ CIV_WORK_DIR=$(pwd)
+ USER_HOME=$(getent passwd $SUDO_USER | cut -d: -f6)
++VSOCK_ID=3
++VM_NAME=civ-sriov
```

Comment:
Standardize and use Android-CIV1?

Reply:
Kept it as Android-CIV as this is always updated by taking launch parameters.